pkg/controller/backup_deletion_controller_test.go

```diff
@@ -201,59 +118,51 @@ func TestBackupDeletionControllerProcessRequest(t *testing.T) {
 				},
 			},
 		}
-		td := setupBackupDeletionControllerTest(t, location, backup)
+		td := setupBackupDeletionControllerTest(t, defaultTestDbr(), location, backup)
 		td.controller.backupStoreGetter = &fakeErrorBackupStoreGetter{}
-		err := td.controller.processRequest(td.req)
+		_, err := td.controller.Reconcile(ctx, td.req)
```

Comment:
Need to change function name from TestBackupDeletionControllerProcessRequest to TestBackupDeletionControllerReconcile